code/_onclick/hud/human.dm

```diff
@@ -38,390 +39,408 @@
 /obj/screen/ling/chems
 	name = "chemical storage"
 	icon_state = "power_display"
+	screen_loc = ui_lingchemdisplay
 
-/datum/hud/proc/human_hud(var/ui_style='icons/mob/screen1_White.dmi', var/ui_color = "#ffffff", var/ui_alpha = 255)
 
-	src.adding = list()
-	src.other = list()
-	src.hotkeybuttons = list() //These can be disabled for hotkey usersx
+/mob/living/carbon/human/proc/remake_hud() //used for preference changes mid-round; can't change hud icons without remaking the hud.
```

Comment:
i made this humans even thought it'll work for all mobs because it shouldn't be used
the HUD shouldn't be deleted and recreated unless completely necessary